Template/LO/SubProcesses/reweight.f

```diff
@@ -1790,6 +1790,7 @@ subroutine update_scale_coupling(p, wgt)
       include 'genps.inc'
       include 'run.inc'
       include 'nexternal.inc'
+      include 'vector.inc'
       include 'coupl.inc'
```

Comment:
same

Reply:
same as above, vector.inc is needed by coupl.inc